sympy/matrices/matrices.py

```diff
@@ -2957,7 +2963,11 @@ def berkowitz(self):
         for i, T in enumerate(transforms):
             polys.append(T*polys[i])
 
-        return tuple(map(tuple, polys))
+        tup = tuple(map(tuple, polys))
+        for i in tup:
+            berk.append(i)
+        return tuple(berk)
```

Comment:
The loop can be avoided by just writing
`` return berk + tuple(map(tuple, polys))``
(assuming berk is a tuple).